core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -229,7 +230,11 @@ public void start(Boolean isPrimary) throws IOException {
         mDailyBackup.start();
       }
     } else {
-      boolean haEnabled = ServerConfiguration.getBoolean(PropertyKey.ZOOKEEPER_ENABLED);
+      boolean haEnabled = ServerConfiguration.getBoolean(PropertyKey.ZOOKEEPER_ENABLED)
```

Comment:
can we use `ConfigurationUtils.isHaMode` for this?

Reply:
Changed to `ConfigurationUtils.isHaMode`, thanks